packages/storybook/src/documentation/pages/demos/columns/extendColTypes.demo.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Columns, RowsProp, XGrid, ColTypeDef } from '@material-ui/x-grid';
+import { randomPrice, randomStatusOptions } from '@material-ui/x-grid-data-generator';
+import '../demo.css';
+
+const highlightCellColType: ColTypeDef = { cellClassName: 'highlight' };
+const usdPriceColType: ColTypeDef = {
+  extendType: 'number',
+  width: 150,
+  valueFormatter: ({ value }) => `${Number(value).toFixed(2)} USD`,
+};
+
+export default function ExtendColTypesDemo() {
+  const columns: Columns = [
```

Comment:
What do you think about moving these variables in the outer scope? It I wonder about the need to create them at each render. In the docs, there are action that can recreate them, so in people's code, maybe that could help performance a bit?

Reply:
Up to you. In the real world, they will be defined inside the component scope.
Performance-wise it's negligible.

Reply:
OK, no preferences, I was assuming that the configuration will 80% of the time be static. But I don't have the whole context on how it's used. In this case, you might want to revert some of my previous changes where I moved the variables outside of the render scope'

Reply:
It seems ok to me. I checked the demos